pom.xml

```diff
@@ -205,7 +205,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>apache-httpcomponents-client-4-api</artifactId>
-      <version>4.5.5-3.0</version>
+      <version>4.5.5-2.1</version>
```

Comment:
Why is the http components client 4 API version going back to an older version?

Reply:
Git-plugin needs that, otherwise enforcer will fail the build.